.dockerignore

```diff
@@ -0,0 +1,34 @@
+# Include any files or directories that you don't want to be copied to your
+# container here (e.g., local build artifacts, temporary files, etc.).
+#
+# For more help, visit the .dockerignore file reference guide at
+# https://docs.docker.com/go/build-context-dockerignore/
+
+**/.DS_Store
+**/__pycache__
```

Comment:
The pattern for excluding `__pycache__` directories should include a leading slash for consistency. Update it to `**/__pycache__/` to ensure it correctly matches all `__pycache__` directories.